client/state/admin-menu/test/selectors.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+import deepFreeze from 'deep-freeze';
+
+/**
+ * Internal dependencies
+ */
+import menuFixture from './menu-fixture.skip';
+import { getAdminMenu } from '../selectors';
+
+const frozenFixture = deepFreeze( menuFixture );
+
+describe( 'selectors', () => {
+	describe( '#getAdminMenu', () => {
+		test( 'returns null when state is undefined', () => {
+			const state = {};
+
+			expect( getAdminMenu( state, 123456 ) ).toEqual( null );
+		} );
+
+		test( 'returns null when siteId is not provided', () => {
+			const state = {};
+
+			expect( getAdminMenu( state ) ).toEqual( null );
+		} );
+
+		test( 'returns null when requested siteId key is not present', () => {
+			const state = {
+				adminMenu: {
+					56789: frozenFixture,
+				},
+			};
+
+			expect( getAdminMenu( state, 12345 ) ).toEqual( null );
+		} );
+
+		test( 'returns menu data when siteId is present', () => {
+			const state = {
+				adminMenu: {
+					56789: frozenFixture,
+					12345: frozenFixture,
+					84649: frozenFixture,
+					95538: frozenFixture,
```

Comment:
shall we make all entries except 12345, `null`?  So that we are sure the correct `frozenFixture` is returned?

Reply:
Yeh that was super silly of me. I've updated so that we use an empty object for all other entires. This ensures the test can fail. Great spot!